### DISCUSSION TITLE
edit global.sls

### DISCUSSION THREAD

**Discussion by mahmoudelhaj** [Security-Onion-Solutions/securityonion#9914] [Unsupported Versions] [answered]
when modifying in global.sls on the manager node to apply it, What is the command to make it apply without restarting?

**Comment by reyesj2** [comment, marked as answer]
Wait 15 minutes for the next highstate to run or manually run ```sudo salt '*' state.highstate```
if you made a change to something specific eg zeek you can do ```sudo salt '*' state.apply zeek```
